xmppserver/src/main/webapp/server-locale.jsp

```diff
@@ -261,6 +261,17 @@
                     <label for="zh_CN">Simplified Chinese (zh_CN)</label>
                 </td>
             </tr>
+             <tr>
+                <td>
+                    <input type="radio" name="localeCode" value="fa" <%= ("fa".equals(locale.toString()) ? "checked" : "") %> id="fa" />
+                </td>
+                <td>
+                    <a href="#" onclick="document.sform.localeCode[1].checked=true; return false;"><img src="images/language_fa.gif" alt="" /></a>
+                </td>
+                <td>
+                    <label for="fa">Persian (fa_IR)</label>
```

Comment:
`fa_IR` should be `fa`, unless the translation is specific to Iranian (in which case all `fa`'s throughout this change should be changed to `fa_IR`).